VeriD/Issues/eventTicket/description.xml

```diff
@@ -62,7 +62,7 @@
 			</Description>
 		</Attribute>
 		<Attribute id="ticket"
-			displayHint="ticket">
+			displayHint="portraitPhoto">
```

Comment:
Apart from displayHint not being used, I want to verify if this is right: You changed `displayHint` at the ticket attribute from `ticket` to `portraitPhoto`?

Reply:
displayHint is used, I tested this with Demo Personal data, if you add a base64 encoded image it displays the image.
So for demo/testing purpose this would be great, would suggest to change the name to "image" later.

Reply:
Ah of course, I'm mistaken. Clear, and agree `image` instead of `portraitPhoto` would be a more generic approach!:)